functions/Test-DbaLastBackup.ps1

```diff
@@ -89,6 +89,15 @@ function Test-DbaLastBackup {
     .PARAMETER Confirm
         If this switch is enabled, you will be prompted for confirmation before executing any operations that change state.
 
+    .PARAMETER MaxTransferSize
+        Parameter to set the unit of transfer. Values must be a multiple of 64kb and a max of 4GB
```

Comment:
If we have specific requirements around the value will the error returned by provided an invalid value be clear to the user?

Reply:
Unsure if maybe the Restore-DbaDatabase command handles that for us, have not tested/checked.

Reply:
in Restore-DbaDatabase this is done:
`            if (($MaxTransferSize % 64kb) -ne 0 -or $MaxTransferSize -gt 4mb) {
                Stop-Function -Category InvalidArgument -Message "MaxTransferSize value must be a multiple of 64kb and no greater than 4MB"
                return
            }
            if ($BlockSize) {
                if ($BlockSize -notin (0.5kb, 1kb, 2kb, 4kb, 8kb, 16kb, 32kb, 64kb)) {
                    Stop-Function -Category InvalidArgument -Message "Block size must be one of 0.5kb,1kb,2kb,4kb,8kb,16kb,32kb,64kb"
                    return
                }
            }`

because of this, and that i only wanted a minimum impact so passthrough style, i did not touch the parameters.
i could copy code above, but that would double that part of code, and not really add anything.

Reply:
Since these are parameters from Restore command we should match the help that command provides as well. So the .PARAMETER for each from the restore command can just be copied/pasted into this one to make them consistent.

Reply:
Tabbing through trying to use the command with the new commands I think it would be more fitting to have these two parameters go after the DeviceType parameter (both in help and param block). That seems to flow better when folks are tabbing through the list.